tests/integration_tests/tests/agentless_tests/test_hooks.py

```diff
@@ -183,15 +186,19 @@ def test_implementation_plugin(self):
         new_config = """
 hooks:
   - event_type: workflow_started
-    implementation: cloudmock.tasks.hook_task
+    implementation: target-aware-mock.tasks.hook_task
     inputs:
       input1: input1_test
       input2: input2_test
     description: test hook
 """
+        upload_mock_plugin('target-aware-mock', '1.0',
```

Comment:
If the arguments can't fit in one line I like it better as separate line for each argument